server/src/main/java/org/elasticsearch/index/mapper/MetadataFieldMapper.java

```diff
@@ -70,6 +70,7 @@ protected MetadataFieldMapper(FieldType fieldType, MappedFieldType mappedFieldTy
      * Called when mapping gets merged. Provides the opportunity to validate other fields a metadata field mapper
      * is supposed to work with before a mapping update is completed.
      */
+    @Override
```

Comment:
We can also add an `@Override` to `DataStreamTimestampMapper#validate`.

Reply:
++